packages/expo-media-library/src/next/index.ts

```diff
@@ -9,15 +9,56 @@ export * from './MediaLibraryNext.types';
 export class Query extends ExpoMediaLibraryNext.Query {}
 
 export class Asset extends ExpoMediaLibraryNext.Asset {
+  /*
+   * Creates a new asset from a given file path.
+   * Optionally associates the asset with an album. On Android, if not specified, the asset will be placed in the default "Pictures" directory.
+   *
+   * @param filePath - Local filesystem path (e.g., `file:///...`) of the file to import.
+   * @param album - Optional `Album` instance to place the asset in.
+   * @returns A promise resolving to the created `Asset`.
```

Comment:
Will this throw an exception if the asset couldn't be created? If so we should mention it.